app/models/container_definition.rb

```diff
@@ -12,7 +12,6 @@ def disconnect_inv
     _log.info "Disconnecting Container definition [#{name}] id [#{id}]"
     self.container.try(:disconnect_inv)
     self.deleted_on = Time.now.utc
-    self.container_group = nil
```

Comment:
I think this involved @moolitayer and @Fryguy in the past (they should be in the loop if they have time to look into this).